docs_src/microservices/application/ApplicationServices.md

```diff
@@ -1,39 +1,55 @@
 
-Typically, after filtering and transforming the data as needed, exporting is the last step in a pipeline to ship the data where it needs to go. There are three primary functions included in the SDK to help facilitate this. The first are the`HTTPPost/HTTPPut` functions that will POST/PUT the provided data to a specified endpoint, and the third is an `MQTTSecretSend()` function that will publish the provided data to an MQTT Broker as specified in the configuration.
+Typically, after filtering and transforming the data as needed, exporting is the last step in a pipeline to ship the data where it needs to go. There are three primary functions included in the SDK to help facilitate this. The first are the`HTTPPost/HTTPPut` functions that will POST/PUT the provided data to a specified endpoint, and the third is an `MQTTSecretSend()` function that will publish the provided data to a MQTT Broker as specified in the configuration.
 
-See [Built-in Functions](./BuiltIn.md) section for full list of SDK supplied functions 
+See [Built-in Pipeline Functions](sdk/api/BuiltInPipelineFunctions.md) section for full list of SDK supplied pipeline functions 
 
 !!! Note
-    The App SDK provides much more functionality than just filtering, formatting and exporting. The above simple example is provided to demonstrate how the functions pipeline works. With the ability to write your custom pipeline functions, your custom application services can do what ever your use case demands.
+    The App SDK provides much more functionality than just filtering, formatting and exporting. The above simple example is provided to demonstrate how the functions pipeline works. With the ability to write your own custom pipeline functions, your custom application services can do what ever your use case demands.
 
-There are three primary triggers that have been included in the SDK that initiate the start of the function pipeline. First is the HTTP Trigger via a POST to the endpoint `/api/{{api_version}}/trigger` with the EdgeX Event data as the body. Second is the EdgeX MessageBus Trigger with connection details as specified in the configuration and the third it the External MQTT Trigger with connection details as specified in the configuration. See the [Triggers](./Triggers.md) section for full list of available `Triggers`
+There are three primary triggers that have been included in the SDK that initiate the start of the function pipeline. First is the HTTP Trigger via a POST to the endpoint `/api/{{api_version}}/trigger` with the message (typically EdgeX Event data) as the body. Second is the EdgeX MessageBus Trigger with connection details as specified in the configuration and the third it the External MQTT Trigger with connection details as specified in the configuration. See the [Triggers](sdk/details/Triggers.md) section for full details on the available triggers.
```

Comment:
Can we move these to bullets or numbers?
```suggestion
There are three primary triggers that have been included in the SDK that initiate the start of the function pipeline. 
1. First is the **HTTP Trigger** via a POST to the endpoint `/api/{{api_version}}/trigger` with the message (typically EdgeX Event data) as the body. 
2. Second is the **EdgeX MessageBus Trigger** with connection details as specified in the configuration.
3. Third is the **External MQTT Trigger** with connection details as specified in the configuration. 
See the [Triggers](sdk/details/Triggers.md) section for full details on the available triggers.
```

Reply:
updated